docs/particles.md

```diff
@@ -112,4 +112,5 @@ when they are in boundary regions are allocated depending on the boundary flags
 in the input file. Currently, outflow and periodic boundaries are supported natively.
 User-specified boundary conditions must be set by specifying the "user" flag in the input
 parameter file and then updating the appropriate Swarm::bounds array entries to separately
-allocated boundary condition objects.
+allocated boundary condition objects. An example is given in the `particles` example when
+ix1 and ox1 are set to `user` in the input parameter file.
```

Comment:
This isn't really accurate right? It might be better to say that we set a factory function which generates the appropriately allocated boundary condition objects. Could that be made a little more explicit in the documentation?

Reply:
Yeah good point, especially since the application-provided cell-centered and particle boundaries are somewhat different (necessary due to how we implement particle boundary condition switching for performance). I'll update.